### DISCUSSION TITLE
Schnittstelle FOXESS nach Victron VRM

### DISCUSSION THREAD

**Discussion by Jaaxy** [nathanmarlor/foxess_modbus#455] [Ideas]
Hallo ich beobachte dieses Projekt schon einige Zeit, habe es persönlich auch mit einem USR-W610 <-> H3 am laufen. Jetzt stehe ich vor einer Thematik das System mit einer Victron MP II 3P zu koppeln. Dazu würde ich gerne die MODBUS Daten direkt in das Victron VRM intergrieren. Es gibt da einen netten Ansatz auf [https://github.com/henne49/dbus-opendtu](url) wo externe Geräte wie z.B. Sonoff Tasmota in das VRM mit eingebunden werden. Eine Kombination aus den beiden Diensten wäre genau das was ich suche.

Per ConfigFile die MODBUS Daten des FOXESS direkt ins Cerbo VRM von Victron.

Ich bin der Meinung das würde die Reichweite beider Communities auch weiter verstärken, wenn die sich direkt unterhalten können. (So wie z.b. Fronius und Victron)

Ich könnte das ganze auch über NodeRed abbilden, da ich in Verbindung mit HA ja zugriff auf den FOX und auf Victron ja sowieso habe. Ich hätte es aber lieber direkt um unnötige Schnittstellen zu vermeiden. 
Dazu fehlen mir aber tiefergehende Kenntnisse in Python.

Vielleicht ist das ja ein Ideenansatz der Resonanz findet

**Comment by canton7** [comment]
Looks interesting.

If anyone uses Victron VRM and wants to take this on as a project I'd be happy to provide support on the FoxESS side.

**Reply by Jaaxy** [reply]
I use a Victron Cerbo with VRM and I have a FOXESS H3 with USR-W610 running. (FOXESS HA plugin)
I would like to make myself available as a tester. However, my programming knowledge is limited to JS, HTML, PHP, CSS.
I haven't been able to gain any knowledge of Python yet.

I'll post the idea on the Victron Modbus Git. Maybe there's someone there with whom we could set this up.
